<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetTemporaryPwr4groups</class>
 <widget class="QDialog" name="SetTemporaryPwr4groups">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>264</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="1">
    <widget class="QLabel" name="lblDtTo">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../res.qrc">
         <normaloff>:/katynko/svg/dialog-cancel.svg</normaloff>:/katynko/svg/dialog-cancel.svg</iconset>
       </property>
       <property name="shortcut">
        <string>Esc</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Apply</string>
       </property>
       <property name="icon">
        <iconset resource="../res.qrc">
         <normaloff>:/katynko/svg/dialog-ok-apply.svg</normaloff>:/katynko/svg/dialog-ok-apply.svg</iconset>
       </property>
       <property name="shortcut">
        <string>Return</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="3">
    <widget class="QSlider" name="hsTimeFrom">
     <property name="maximum">
      <number>1439</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="lblDtFrom">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QTimeEdit" name="timeEdit"/>
   </item>
   <item row="5" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>25</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="2">
    <widget class="QTimeEdit" name="timeEdit_2"/>
   </item>
   <item row="1" column="2">
    <widget class="QSpinBox" name="spinBox">
     <property name="minimum">
      <number>9</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="4">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Temporary schedule for group IDs</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblTempSchedule">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="3">
    <widget class="QSlider" name="hsTimeTo">
     <property name="maximum">
      <number>1439</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QSlider" name="hsPower">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>To</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Linear Power [%]</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>From</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
